app/static/src/app/store/sensitivity/mutations.ts

```diff
@@ -88,5 +89,15 @@ export const mutations: MutationTree<SensitivityState> = {
     },
     [SensitivityMutation.ParameterSetDeleted](state: SensitivityState, parameterSetName: string) {
         delete state.parameterSetResults[parameterSetName];
+    },
+
+    [SensitivityMutation.ParameterSetSwapped](state: SensitivityState, parameterSetName: string) {
+        const temp = state.result;
+        state.result = state.parameterSetResults[parameterSetName] || null;
+        if (temp == null) {
```

Comment:
`===` here? or `!temp` if you want relaxed comparison?

Reply:
ahh yes I wanted relaxed comparison in case temp is undefined somehow since undefined == null, can change it to !temp

Reply:
I think with TypeScript a lot of that defensive programming is less important but @EmmaLRussell will have much better insight than me on that

Reply:
all done

Reply:
Yeah, I think falsy check here is the way to go